src/SignalRServiceExtension/Config/SignalRConfigProvider.cs

```diff
@@ -4,19 +4,23 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
 using Microsoft.Azure.SignalR.Management;
 using Microsoft.Azure.WebJobs.Description;
 using Microsoft.Azure.WebJobs.Host.Bindings;
 using Microsoft.Azure.WebJobs.Host.Config;
+using Microsoft.Azure.WebJobs.Logging;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
 {
-    [Extension("SignalR")]
-    internal class SignalRConfigProvider : IExtensionConfigProvider
+    [Extension("SignalR", "signalr")]
+    internal class SignalRConfigProvider : IExtensionConfigProvider, IAsyncConverter<HttpRequestMessage, HttpResponseMessage>
```

Comment:
Why are we using HttpRequestMessage and HttpResponseMessage for everything?

Reply:
It's Azure Function webhook provider's requirement. It pass HttpRequestMessage to us and receive HttpResponseMessage as return.

Reply:
Can we also support HttpRequest? I see HttpTrigger support both

Reply:
@wanlwanl  As far as I know, `HttpTrigger` is so different that it highly integrate with Azure Function. So, `HttpTrigger` has the "privilege" over other triggers. `HttpTrigger` can design the route path, but if we use webhookProvider, we need keep there convension and the url path must be "host/runtime/webhook/signalr".